File: src/SextantTG/SextantTG.WebServices/DTO/SubtourItem.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SextantTG.WebServices.DTO
{
    public class SubtourItem : DTO
    {        
        /// <summary>
        /// 所属旅行ID
        /// </summary>
        public string TourId { get; set; }

        /// <summary>
        /// 子旅行ID
        /// </summary>
        public string SubtourId { get; set; }

        /// <summary>
        /// 子旅行名称
        /// </summary>
        public string SubtourName { get; set; }

        /// <summary>
        /// 相关景点ID
        /// </summary>
        public string SightId { get; set; }

        /// <summary>
        /// 相关景点
        /// </summary>
        public string SightName { get; set; }

        /// <summary>
        /// 起始时间
        /// </summary>
        public DateTime BeginDate { get; set; }

        /// <summary>
        /// 结束时间
        /// </summary>
        public DateTime EndDate { get; set; }
    }
}

File: src/App/src/com/ss/stg/dto/SubtourItem.java
package com.ss.stg.dto;

import java.util.Date;

/**
 * 子旅行
 * 
 * @author ggxx
 * 
 */
public class SubtourItem extends DTO {

	public SubtourItem() {
	}

	private String tourId;
	private String subtourId;
	private String subtourName;
	private String sightId;
	private String sightName;
	private Date beginDate;
	private Date endDate;

	public String getTourId() {
		return tourId;
	}

	public void setTourId(String tourId) {
		this.tourId = tourId;
	}

	public String getSubtourId() {
		return subtourId;
	}

	public void setSubtourId(String subtourId) {
		this.subtourId = subtourId;
	}

	public String getSubtourName() {
		return subtourName;
	}

	public void setSubtourName(String subtourName) {
		this.subtourName = subtourName;
	}

	public String getSightName() {
		return sightName;
	}

	public void setSightName(String sightName) {
		this.sightName = sightName;
	}

	public Date getBeginDate() {
		return beginDate;
	}

	public void setBeginDate(Date beginDate) {
		this.beginDate = beginDate;
	}

	public Date getEndDate() {
		return endDate;
	}

	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}

	public String getSightId() {
		return sightId;
	}

	public void setSightId(String sightId) {
		this.sightId = sightId;
	}
}
